core/templates/xregistry_openapi_template.json

```diff
@@ -1,1068 +1,1937 @@
+            },
+            "filter": {
+                "name": "filter",
+                "in": "query",
+                "required": false,
+                "explode": true,
+                "schema": {
+                    "type": "array",
+                    "items": { "type": "string" }
+                },
+                "description": "Filter criterion(s) (see spec: Filter)."
+            },
+            "doc": {
+                "name": "doc",
+                "in": "query",
+                "required": false,
+                "schema": { "type": "boolean" },
+                "description": "If true, returns the document view (with relative URLs and default version data removed). See spec: Doc."
             }
-          },
-          "400": {
-            "description": "Bad Request - constraint failure"
-          },
-          "409": {
-            "description": "Conflict - epoch not greater"
-          }
-        }
-      }      
-    },
-    "/{%-groupNamePlural-%}/{groupid}/{%-resourceNamePlural-%}/{resourceid}": {
-      "parameters": [
-        {
-          "$ref": "#/components/parameters/groupid"
-        },
-        {
-          "$ref": "#/components/parameters/resourceid"
         },
-        {
-          "in": "query",
-          "name": "meta",
-          "description": "Interact with the metadata",
-          "required": false,
-          "schema": {
-            "type": "boolean"
-          }
-        }
-      ],
-      "get": {
-        "operationId": "get{%-resourceNameSingular-%}",
-        "responses": {
-          "200": {
-            "description": "The corresponding resource",
-            "headers": {
-              "resource-id": {
-                "$ref": "#/components/headers/resource-id"
-              },
-              "resource-version": {
-                "$ref": "#/components/headers/resource-version"
-              },
-              "resource-name": {
-                "$ref": "#/components/headers/resource-name"
-              },
-              "resource-self": {
-                "$ref": "#/components/headers/resource-self"
-              },
-              "resource-description": {
-                "$ref": "#/components/headers/resource-description"
-              },
-              "resource-documentation": {
-                "$ref": "#/components/headers/resource-documentation"
-              },
-              "resource-labels": {
-                "$ref": "#/components/headers/resource-labels"
-              },
-              "resource-createdat": {
-                "$ref": "#/components/headers/resource-createdat"
-              },
-              "resource-modifiedat": {
-                "$ref": "#/components/headers/resource-modifiedat"
-              },
-              "Content-Location": {
-                "description": "permalink location of the returned version",
+        "headers": {
+            "resource-id": {
```

Comment:
xRegistry-xxx
I don't think we need %-encoding since we have strict restrictions on the chars